### ISSUE TITLE
Stereochemistry being lost on spiro ring system

### ISSUE THREAD

**Issue by coleb** [rdkit/rdkit#1412]
```python
>>> Chem.MolToSmiles(Chem.MolFromSmiles('C[C@H]1CC[C@@]2(CC1)C=C(C)C(=C2)'), True)
'CC1=CC2(C=C1)CCC(C)CC2'
```

The sanitization is stripping out this stereo center:
![image](https://cloud.githubusercontent.com/assets/127899/25966498/2da82a50-3659-11e7-8db9-92ccdcc7b6b2.png)

Was expecting the stereo remain like this: 

```python
>>> Chem.MolFromSmiles('C[C@H]1CC[C@@]2(CC1)C=C(C)C(=C2)', sanitize=False)
```

![image](https://cloud.githubusercontent.com/assets/127899/25966550/565abce2-3659-11e7-951c-42a8e68eaa38.png)

**Comment by proteneer** [comment]
isomericSmiles=True

**Comment by coleb** [comment]
Copy and paste error, isomericSmiles=True still loses it. 

```python
>>>Chem.MolToSmiles(Chem.MolFromSmiles('C[C@H]1CC[C@@]2(CC1)C=C(C)C(=C2)'), True)
'CC1=CC2(C=C1)CCC(C)CC2'
```

**Comment by proteneer** [comment]
This is actually more like cis-trans isomerism rather than true diastereomerism.

Given the two chiral carbons, A and B, there are four possible permutations:

A@ B@ -Y1
A@@ B@ - X1
A@ B@@ - X2
A@@ B@@ - Y2

Of these four possible permutations, there are only two distinct molecules X and Y (both methyls point in the same direction, vs pointing in opposite directions). Oddly enough I hope someone who knows CIP rules well enough can tell me if they should be assigned R or S :)

So you're right that the canonicalizer should be consistent, and right now it's incorrect.

**Comment by greglandrum** [comment]
Are you sure you're using the most recent version of the code? This was fixed as part of #1294 
```
In [2]: Chem.CanonSmiles('C[C@H]1CC[C@@]2(CC1)C=C(C)C(=C2)')
Out[2]: 'CC1=C[C@]2(C=C1)CC[C@@H](C)CC2'
```

**Comment by coleb** [comment]
Ah, you're correct, it was indeed fixed in the latest version. Closing then.
